ProcessMaker/processmaker
# As a process architect I want to have documentation of the User personal access token API, so I can manage user personal tokens

## Issue & Reproduction Steps
As a process architect I want to have documentation of the User personal access token API, so I can manage user personal tokens

## Solution
- Add missing API documentation for user tokens.

## How to Test
1. Navigate to `/api/documentation` in your local instance.
2. Authorize yourself via Bearer Token
3. Locate the 'Personal Tokens' location
4.  'Try Out' all APIs
5. Ensure all fields work properly
6. Ensure all properties are listed in the example schema

## Related Tickets & Packages
- [FOUR-5980](https://processmaker.atlassian.net/browse/FOUR-5980)

## Code Review Checklist
